### PR TITLE
Create Formulaic Privacy Notice

### PR DESCRIPTION
**Pull request by aroybalreid**
Privacy Notice for Formulaic product

We can host this on this page https://www.mozilla.org/en-US/privacy/.  We will also want a URL that the Formulaic Product team can link to

### CODE REVIEW

File: en/Formulaic Privacy Notice [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Formulaic Privacy Notice
+
+Last Updated March 8, 2024
+{: datetime="2024-03-08" }
+
+This privacy notice applies to Formulaic, a service that allows users to query, create, share, and manage templates for querying artificial intelligence (AI) based large language models (LLMs) (the “Service”). The Service is owned, licensed, and operated by Mozilla Corporation (referred to herein as "we," "us," "our," or "Mozilla"). 
+This privacy notice explains what information Formulaic collects, shares, and processes, and why. We also adhere to the [Mozilla Privacy Policy](https://www.mozilla.org/privacy/) when we receive, handle, and share personal information. Your use of the Service constitutes your acceptance of this privacy notice, which incorporates by reference our Terms of Service.
```
Comment (peiying2) [inline en/Formulaic Privacy Notice:7]: Insert a line space

File: en/Formulaic Privacy Notice [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+**Communications.** When you contact us in connection with the Service, we may also collect personal information, such as your username, name, email address, or phone number, as well as any other personal information you may provide to us.  
+
+We also *automatically* receive certain information when you interact with the Service: 
+**Technical data.** We receive basic telemetry data by default, which we use to improve the performance, stability, and security of the Service, including information about your device, including hardware configuration and device operating system; browser information, such as browser type and settings; log files and usage data; and event information, such as errors or crashes. We may also receive additional information about your device, such as country, language, operator and OEM, which we use to prevent fraud and abuse, enforce our terms, and improve the security of the Service.
```
Comment (peiying2) [inline en/Formulaic Privacy Notice:19]: Insert a line break

File: en/Formulaic Privacy Notice [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+{: datetime="2024-03-08" }
+
+This privacy notice applies to Formulaic, a service that allows users to query, create, share, and manage templates for querying artificial intelligence (AI) based large language models (LLMs) (the “Service”). The Service is owned, licensed, and operated by Mozilla Corporation (referred to herein as "we," "us," "our," or "Mozilla"). 
+This privacy notice explains what information Formulaic collects, shares, and processes, and why. We also adhere to the [Mozilla Privacy Policy](https://www.mozilla.org/privacy/) when we receive, handle, and share personal information. Your use of the Service constitutes your acceptance of this privacy notice, which incorporates by reference our Terms of Service.
+
+##Things you should know
+We collect the following information directly from you when you use the Service:
+**Username.** This is a unique, public-facing account name chosen by you. Any content that you choose to publish on the Formulaic website or otherwise contribute to the Service will include this username. We use this username to attribute you with any content you contribute or publish when using the Service, as well as to enforce our Content Policies.
```
Comment (peiying2) [inline en/Formulaic Privacy Notice:11]: Insert a line break

File: en/Formulaic Privacy Notice [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+**Interaction data.** We receive aggregate data about visits to the Formulaic website and other aggregate insights about user interactions with the Service. We use this data to better understand usage of the Service, as well as to add, improve, or remove certain features of the Service. We may also receive HTTP referral data to understand the website or domain that referred you to our site, which we only use for purposes of addressing bugs or other technical issues with our Service. We automatically delete this data after 30 days.   
+**Metrics.** We may also use cookies, device information, and IP addresses, along with clear GIFs, cookies, and third party services to securely operate the Service, help us understand how users engage with our Service, such as visits to our site and settings preferences, and provide additional functionality. We use:
+-Sign in with Google, which places a cookie on your device to assist with sign-in and authentication to the Service. When you sign into the Service, you accept the associated [Privacy Policy](https://policies.google.com/privacy?hl=en-US) and [Terms of Service](https://policies.google.com/terms) from Google.
```
Comment (peiying2) [inline en/Formulaic Privacy Notice:23]: - Line break before the listed items.
- Remove en-US in the URL: `https://policies.google.com/privacy`

File: en/Formulaic Privacy Notice [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+-Google Analytics to obtain metrics on how users engage with our websites. This helps us to improve site content. For more information about how Google uses your personal information, please visit [Google Analytics’ Privacy & Security](https://support.google.com/analytics/topic/2919631?hl=en&ref_topic=1008008&sjid=14989286036636170427-NA). You can install the [Google Analytics Opt-out Browser Add-on](https://tools.google.com/dlpage/gaoptout) to prevent data collection about your visits to the Service and prohibit data transmission to Google Analytics.
```
Comment (peiying2) [inline en/Formulaic Privacy Notice:24]: Can we remove `en` in the link to automatically redirect to default language of choice?

File: en/Formulaic Privacy Notice [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+We also receive the following *information from third parties* when you use the Service: 
+**Account data.** You need a Google Account in order to create an account and access the Service. When you log in with your Google Account to access the Service, we receive your name, email address, language preference, and profile picture from Google, and use this information to assist with your account administration and to contact you in relation to the Service, as well as to enforce our Terms of Service and Content Policies. Use of a Google Account is governed by [Google's Privacy Policy](https://policies.google.com/privacy) and [Terms of Service](https://policies.google.com/terms).
```
Comment (peiying2) [inline en/Formulaic Privacy Notice:27]: Insert a line break before bullet items.

File: en/Formulaic Privacy Notice [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+**Reports of violations.** The Service also allows users to report suspected violations of our Content Policies, which are reviewed by human moderators. If we receive a report or complaint about any Formula(s) that you have published, we will retain any personal information shared with us in connection with that report or complaint for as long as we need to review, adjudicate, and resolve the case, or as otherwise required by law.   
+
+We may also use information about you to find and address violations of our Terms of Service or Content Policies; investigate suspicious activity; detect, prevent, and combat harmful or unlawful behavior; and otherwise maintain the integrity of our Service.
+
+##Information We Share. 
+We use third parties to provide the Service to you, and have contracted with these companies requiring them to protect your information (Third-Party Services):
```
Comment (peiying2) [inline en/Formulaic Privacy Notice:35]: Line break between heading and paragraph

File: en/Formulaic Privacy Notice [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+**Google Cloud Provider.** Google Cloud Provider (GCP) is a cloud-computing platform. We use GCP to facilitate account creation, authenticate users, host content, and provide access to certain Third-Party Models available through the [Google Vertex](https://cloud.google.com/vertex-ai/generative-ai/docs/learn/overview) service.
```
Comment (peiying2) [inline en/Formulaic Privacy Notice:36]: Line break before bullet items.

File: en/Formulaic Privacy Notice [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+**Anyscale.** Anyscale is a cloud provider that offers scaled hosting and inference of open source LLMs. We use Anyscale to provide access to certain open source Third-Party Models via our Service.
+**OpenAI.** OpenAI is an AI research and deployment company that offers certain LLMs for individual and enterprise use, APIs, and other services. We use OpenAI to provide access to certain Third-Party Models, including ChatGPT, via our Service.
+
+Third-Party Services, including any Outputs generated by the use of Third-Party Services and Third-Party Models, are subject to their own terms, and we are not responsible for them.
+
+##Children
+The Service is not directed to children under 18 years of age, and we do not knowingly collect or solicit personal information from children. If the Service unknowingly collects personal information from a child under the age of 18 without parental consent, we will delete that information as soon as possible, unless we have a legal obligation to keep it. If you are a parent or guardian and believe your child has uploaded personal information to our site without your consent, you may contact us as described in “How to Contact Us” below.
```
Comment (peiying2) [inline en/Formulaic Privacy Notice:43]: Insert line break between heading and first paragraph.

**Review by peiying2 — CHANGES_REQUESTED** [review]
Did not mark up all the formatting related issues. 
- Line break between heading and first paragraph
- Link break before listed or bullet items.
- URL: remove `en` or `en-US` in links

File: en/Formulaic Privacy Notice [hunk 10]
```diff
@@ -0,0 +1,77 @@
+# Formulaic Privacy Notice
+
+Last Updated March 8, 2024
+{: datetime="2024-03-08" }
+
+This privacy notice applies to Formulaic, a service that allows users to query, create, share, and manage templates for querying artificial intelligence (AI) based large language models (LLMs) (the “Service”). The Service is owned, licensed, and operated by Mozilla Corporation (referred to herein as "we," "us," "our," or "Mozilla").
```
Comment (peiying2) [inline en/Formulaic Privacy Notice:6]: Use curly quotes (not straight quotes) as in `“Service”` for the other ones in this paragraph

**Comment by flodolo** [comment]
File preview here: https://github.com/mozilla/legal-docs/blob/0cfbb4178da1f8b4580a956776174d1d79937bc9/en/formulaic_privacy_notice.md

**Comment by aroybalreid** [comment]
I see some formatting issues.  Is there a way for me to edit at this point?  It is not clear that I can do so right now

**Review by peiying2 — APPROVED** [review]

**Review by flodolo — APPROVED** [review]

**Pull request merged**